src/style.css

```diff
@@ -1,3 +1,74 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
```

Comment:
You have used a specific font, but there is no fallback font. In case the main font doesn't work, the browser will choose a system font, which may not match with your design. It's always a good practice to specify a fallback font. For example, you could use a generic font family such as 'sans-serif' as a fallback.